plugins/button/plugin.js

```diff
@@ -298,7 +298,7 @@
 				title: this.title + ( shortcut ? ' (' + shortcut.display + ')' : '' ),
 				ariaShortcut: shortcut ? editor.lang.common.keyboardShortcut + ' ' + shortcut.aria : '',
 				titleJs: env.gecko && !env.hc ? '' : ( this.title || '' ).replace( "'", '' ),
-				hasArrow: this.hasArrow ? 'true' : 'false',
+				hasArrow: this.hasArrow || false,
```

Comment:
Please note that attribute should get `'false'` (string with value "false"), not a real boolean. The same is true for `true` value.